clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -427,7 +427,7 @@ void visualstudio::Linker::ConstructJob(Compilation &C, const JobAction &JA,
 MSVCToolChain::MSVCToolChain(const Driver &D, const llvm::Triple &Triple,
                              const ArgList &Args)
     : ToolChain(D, Triple, Args), CudaInstallation(D, Triple, Args),
-      RocmInstallation(D, Triple, Args) {
+      RocmInstallation(D, Triple, Args), SYCLInstallation(D) {
```

Comment:
Why do we not pass Triple and Args for SYCL Installation?

Reply:
The `Triple` and `Args` are not used with our installation setup, so there is no need to add them.